<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="in-madhurvk-1" name="apd.infoimage.iwm.tests.INFO_5111" tests="2" failures="0" timestamp="06 Dec 2019 05:45:44 GMT" time="239.073" errors="1">
  <testcase name="testRetrieveAndReserve" time="239.073" classname="apd.infoimage.iwm.tests.INFO_5111">
    <error type="atu.testng.reports.exceptions.ATUReporterStepFailedException">
      <![CDATA[[ATU Custom Reporter Step Failed Exception]
at atu.testng.reports.ATUReports.stepFailureHandler(Unknown Source)
at atu.testng.reports.ATUReports.add(Unknown Source)
at apd.infoimage.iwm.tests.INFO_5111.testRetrieveAndReserve(INFO_5111.java:137)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testRetrieveAndReserve -->
  <testcase name="testRetrieveReserveAndSend" classname="apd.infoimage.iwm.tests.INFO_5111">
    <skipped/>
  </testcase> <!-- testRetrieveReserveAndSend -->
</testsuite> <!-- apd.infoimage.iwm.tests.INFO_5111 -->
